prac_07/squaring.kv

```diff
@@ -0,0 +1,19 @@
+BoxLayout:
+    orientation: 'vertical'
+    BoxLayout:
+        TextInput:
+            id: input_number
+            text: '7'
+            font_size: 48
+            multiline: False
+        Button:
+            text: 'Square'
+            # the following line specifies the function in the app class to call when the button is pressed
+            on_press: app.handle_calculate(int(input_number.text))
+        Label:
+            id: output_label
+            font_size: 48
+            color: (255/255, 192/255, 203/255, 1)
+    BoxLayout:
```

Comment:
This doesn't need to exist - only need a layout widget to contain multiple other widgets.